smithy-kotlin-codegen/src/main/kotlin/software/amazon/smithy/kotlin/codegen/CodegenVisitor.kt

```diff
@@ -22,7 +22,7 @@ import software.amazon.smithy.model.Model
 import software.amazon.smithy.model.knowledge.ServiceIndex
 import software.amazon.smithy.model.neighbor.Walker
 import software.amazon.smithy.model.shapes.*
-import software.amazon.smithy.model.traits.EnumTrait
+import software.amazon.smithy.model.traits.* // Enum Trait
```

Comment:
Kotlin doesn't allow `@Suppress("DEPRECATION")` on import statements, so using wildcard import as a workaround.

Reply:
That's true, but instead of doing the wildcard-import workaround, we should just fully qualify the type at the usage site.

eg. in this file:
```kotlin
if (shape.hasTrait<@Suppress("DEPRECATION") software.amazon.smithy.model.traits.EnumTrait>()) {
    // ...
```